docs/deployment/build-id.md

```diff
@@ -1,13 +1,15 @@
 # Understand: Build ids
 
-We use a build id to define versions for *Reactive Trader Cloud*. This *build id* is a string. It could be a number or a keyword. The *build id* is first used when we *build* to name the docker images. It then can be used to run or deploy that specific reactive trader cloud version.
+We use a build id to define versions for *Reactive Trader Cloud*. This *build id* is a string. It could be a number or a keyword. The *build id* is first used when we *build* to name the reactive trader cloud docker images. It then can be used to run or deploy that specific reactive trader cloud version.
```

Comment:
Since it's open source project, I would change all 'rtc' to 'RTC' and all 'reactive trader cloud' to 'Reactive Trader Cloud' - everywhere in the documentation (find and replace :) )

Reply:
In next commit